doc/source/whatsnew/v2.2.0.rst

```diff
@@ -252,6 +252,7 @@ Bug fixes
 - Bug in :class:`pandas.core.window.Rolling` where duplicate datetimelike indexes are treated as consecutive rather than equal with ``closed='left'`` and ``closed='neither'`` (:issue:`20712`)
 - Bug in :meth:`DataFrame.apply` where passing ``raw=True`` ignored ``args`` passed to the applied function (:issue:`55009`)
 - Bug in :meth:`pandas.read_excel` with a ODS file without cached formatted cell for float values (:issue:`55219`)
+- Bug in :meth:`pandas.read_excel` where ODS files with comments on time value cells failed to parse (related to :issue:`55200`)
```

Comment:
```suggestion
- Bug in :meth:`pandas.read_excel` where ODS files with comments on time value cells failed to parse (:issue:`55200`)
```

Reply:
Thanks - applied, assuming it's clear enough that while this PR takes care of comments on *time* values #55200 targets *string* values and requires a separate fix